test/complexity_measures/statistical_complexity.jl

```diff
@@ -0,0 +1,20 @@
+using Distances
+
+@testset "Statistical Complexity" begin
+    x = randn(1000)
+    est = SymbolicPermutation(; m=3, τ=1)
+    @test statistical_complexity(x, est) isa Real
+    # the complexity should be normalized
+    @test 0.0 <= statistical_complexity(x, est) <= 1.0
```

Comment:
We should strive for testing edge cases explicitly. The normalized complexity is on `[0, 1]`. Below, you construct an example where the statistical complexity is 0.0. Can you make an example (or get it from the original publication) where the normalized statistical complexity is exactly or approaches 1.0? This is not crucial, for completeness of the test suite, it would be nice to have such an example.

Reply:
The way this complexity is defined, it's never actually `1`. This is because it's the _normalized_ JS divergence (this will between 0 and 1), which is then multiplied by the normalized PE. So, the maximum (and minimum) possible complexity is actually a function of the PE, for a given length of patterns `m`. I attached a screenshot from the original paper, maybe this makes clarifies a bit what I mean. The black curves in the plot are the maximum and minimum complexity curves.

![image](https://user-images.githubusercontent.com/69455494/196361075-95cd4742-95a5-44be-ade3-2025b330125d.png)

I can of course test that the statistical complexity is smaller than the maximum value for a given PE once I've implemented the calculation of the curves and tested this separately.

Reply:
> The way this complexity is defined, it's never actually 1

Ah, ok! Then just change the condition to check that the complexity is larger than zero.

> I can of course test that the statistical complexity is smaller than the maximum value for a given PE once I've implemented the calculation of the curves and tested this separately.

Sure, sounds good!